static/app/views/organizationContextContainer.tsx

```diff
@@ -311,6 +316,7 @@ class OrganizationContextContainer extends Component<Props, State> {
         <OrganizationContext.Provider value={this.state.organization}>
```

Comment:
do we still need line 231-233 then?

Reply:
Yeah migrating over the hook in getsentry